### PR TITLE
Bug 1182071 - Allow System app to fetch Sync keys. r=kgrandon

### PR DESCRIPTION
**Pull request by ferjm**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=716f33f11ea652e26e8b5ec16b3f36a9747ba507

File: apps/system/js/fx_accounts_client.js [hunk 1]
```diff
@@ -82,6 +82,12 @@ var FxAccountsClient = function FxAccountsClient() {
     }, successCb, errorCb);
   };
 
+  var getKeys = function getKeys(successCb, errorCb) {
```
Comment (KevinGrandon) [inline apps/system/js/fx_accounts_client.js]: I would prefer to make this a promise instead, is that possible?

Comment (KevinGrandon) [inline apps/system/js/fx_accounts_client.js]: Well after looking again it appears this entire file is callback based, so probably fine for now. Would be nice in the future though.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e33b0745e27635003f277cc494add7986feebb75

**Pull request merged**